src/platform_impl/macos/window_delegate.rs

```diff
@@ -74,6 +74,9 @@ impl Default for PlatformSpecificWindowAttributes {
 
 #[derive(Debug)]
 pub(crate) struct State {
+    /// Strong reference to the global application state.
+    app_delegate: Id<ApplicationDelegate>,
```

Comment:
Does it matter in which order it's dropped?

Reply:
Nope